README.md

```diff
@@ -239,10 +239,6 @@ OPTIONS:
    --s3.region value The AWS region. Required when not specifying S3/minio
       access keys. [$BAZEL_REMOTE_S3_REGION]
 
-   --s3.key_version value Set to 1 for the legacy flat key format, or 2 for
-      the newer format that reduces the impact of S3 rate limits. (default: 1)
-      [$BAZEL_REMOTE_S3_KEY_VERSION]
-
```

Comment:
Could you regenerate the help text, so the "DEPRECATED" note appears here? I used `COLUMNS=80 ./bazel-remote -h` to do this last time.